C/plugins/storage/sqlite/common/readings.cpp

```diff
@@ -1614,7 +1614,7 @@ vector<string>  asset_codes;
 						sql_cmd_base += ",  asset_code";
 
 					sql_cmd_base += ", id, reading, user_ts, ts ";
-					StringReplaceAll (sql_cmd_base, "asset_code", " \"_assetcode_\" .assetcode. ");
+					StringReplaceAll (sql_cmd_base, "asset_code", " '_assetcode_' .assetcode. ");
```

Comment:
This is causing issues now with `''`.

Reply:
Please test for following:

sin"1
sin"1"
sin'1
sin'1'
sin "#1'
sin :."',."''""`/1

Reply:
failed for sin"1, sin"1", sin'1